### PR TITLE
🌐 Add Chinese translation for Advanced - Custom Response

### PR DESCRIPTION
**Pull request by RunningIkkyu**
Add Chinese docs for advance/custom-response.md 
Please help review. @waynerv  @Dustyposa

### CODE REVIEW

File: docs/zh/docs/advanced/custom-response.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
+# 自定义响应 - HTML，流，文件和其他
+
+**FastAPI** 默认会使用 `JSONResponse` 返回响应。
+
+你可以通过直接返回 `Response` 来重载它，参见 [直接返回响应](response-directly.md){.internal-link target=_blank}。
+
+但你可以直接返回 `Response`，返回数据不会自动转换，也不会自动生成文档（例如，在 HTTP 头 `Content-Type` 中包含特定的「媒体类型」作为生成的 OpenAPI 的一部分）。
```
Comment (waynerv) [inline docs/zh/docs/advanced/custom-response.md:7]: missing the word "if",should be "但如果你直接返回一个 `Response`"

File: docs/zh/docs/advanced/custom-response.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+你还可以在 *路径操作装饰器* 中声明你想用的 `Response`。
+
+你从 *路径操作函数* 中返回的内容将被放在该 `Response` 中。
+
+并且如果该 `Response` 有一个 JSON 媒体类型（`application/json`），比如使用 `JSONResponse` 或者 `UJSONResponse` 的时候，你返回的数据会自动转换成任何你在 *路径操作装饰器* 中声明的 Pydantic 的 `response_model` 类型。
```
Comment (waynerv) [inline docs/zh/docs/advanced/custom-response.md:13]: Missing words "(and filtered) ".And "converted with" might means "using Pydantic response_model to converte" not "convert to"

File: docs/zh/docs/advanced/custom-response.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    `ORJSONResponse` 目前只在 FastAPI 中可用，而在 Starlette 中不可用。
+
+
+
+## HTML 响应
+
+使用 `HTMLResponse` 来从 **FastAPI** 中直接返回一个 HTML 响应。
+
+* 导入 `HTMLResponse`。
+* 将 `HTMLResponse` 作为你的 *路径操作* 的 `content_type` 参数传入。
```
Comment (waynerv) [inline docs/zh/docs/advanced/custom-response.md:45]: There is a mistake in english docs,the `content_type` here should be "response_class"

File: docs/zh/docs/advanced/custom-response.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+!!! warning "警告"
+    *路径操作函数* 直接返回的 `Response` 不会被 OpenAPI 的文档记录（比如，`Content-Type` 不会被文档记录），并且在自动化交互文档中也是不可见的。
+
+!!! info "提示"
+    当然，实际的 `Content-Type` 头，状态码等等，将来自于你返回的 `Response` 对象。
+
+### OpenAPI 中的文档和重载 `Response`
+
+如果你想要在函数内容重载响应，但是同时在 OpenAPI 中文档化「媒体类型」，你可以使用 `response_class` 参数并返回一个 `Response` 对象。
```
Comment (waynerv) [inline docs/zh/docs/advanced/custom-response.md:76]: "内容“ should be "内".

File: docs/zh/docs/advanced/custom-response.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+
+接着`response_class` 参数只会被用来文档化 OpenAPI 的 *路径操作*，你的 `Response` 也是如此。
```
Comment (waynerv) [inline docs/zh/docs/advanced/custom-response.md:78]: 你的 `Response` 用来返回响应

**Review by waynerv — CHANGES_REQUESTED** [review]

**Comment by RunningIkkyu** [comment]
@Dustyposa @phy25 @yanever @Laineyzhang55  @waynerv Please take a look.

**Review by waynerv — APPROVED** [review]

**Review by Laineyzhang55 — APPROVED** [review]

**Comment by tiangolo** [comment]
Thank you @RunningIkkyu ! :rocket: :nerd_face: 

And thanks again for the reviews @waynerv and @Laineyzhang55 :bowing_man: :clap:

**Pull request merged**